scripts/postbuild.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# This script prepares the `build` directory for NPM package creation.
+# It first copies all non-code files into the `build` directory, including `package.json` which
+# is edited via a few `sed` commands. These edits include corrections of paths (e.g. entry points)
+# in the copied package.json so that they align with the directory structure inside `build`.
+
+BUILD_DIR=build
+CJS_DIR=dist
+ESM_DIR=esm
+TYPES_DIR=types
+
+ASSETS="README.md
+LICENSE
+package.json"
+
+# check if build dir exists
+if [ ! -d $BUILD_DIR ]; then
+  echo "Directory ${BUILD_DIR}/ DOES NOT exist."
+  echo "This script should only be executed after you've run \`yarn build\`."
+  exit 1
+fi
+
+# copy non-code assets to build dir
+for f in $ASSETS; do
+  cp $f $BUILD_DIR/
+done
+
+# TEMPORARY (TODO remove in v7) copy types to esm and dist
+cp -r $BUILD_DIR/$TYPES_DIR/ $BUILD_DIR/$CJS_DIR
+cp -r $BUILD_DIR/$TYPES_DIR/ $BUILD_DIR/$ESM_DIR
+
+# package.json modifications
+
+# sed command to modify package.json entry points in build dir
+# remove `BUILD_DIR` from `main`, `module` and `type` entry point paths
+entryPointsCommand="/\"(main|module|types)\": .*,/s/$BUILD_DIR\///"
+
+# modify volta extends path
+# remove one `../` from the extension path of volta
+voltaExtendsCommand="/\"extends\": \"\.\.\/\.\.\/\.\.\/package.json\"/s/\.\.\/\.\.\/\.\.\//\.\.\/\.\.\//"
```

Comment:
```suggestion
# sed command to modify volta extends path
# remove one `../` from the extension path of volta
voltaExtendsCommand="/\"extends\": \"\.\.\/\.\.\/\.\.\/package.json\"/s/\.\.\/\.\.\/\.\.\//\.\.\/\.\.\//"
```

My internal monologue: _Wait, I'm not quite sure I get what's happening here. Is he setting a variable or running `sed`? Ohhhh, this is a **sed** command. That's all inside the string. Riiiiiiiiight. Okay, I get it._

(When I first looked at the file, my eye was drawn immediately to the backslash-and-period extravaganza happening on this line, and so I hadn't yet read the comment above `entryPointsCommand`. Whoops. Still, can't hurt to be crystal clear.)

Reply:
I know, this regex is reaaally hard to read. Was sort of cringing as I was writing it out but I couldn't come up with a cleaner solution. Totally open for suggestions though.

Reply:
I think the regex is what it is. I definitely think the second comment is needed, though!

EDIT: Hang on, though. Are we even sure we need this? What happens if you just delete the volta config all together? Now that I'm thinking about it, its only job is to tell volta which versions of node and yarn should be used when working in `sentry-javascript`, not when using `@sentry/browser`.

Reply:
I went with not touching it at all for now, as it didn't seem to make a difference. I was thinking that the volta path needed adjustment for the `pack` script but it seemed like that wasn't necessary. Good catch, thanks